projects/core/src/occ/adapters/user/occ-customer-coupon.adapter.spec.ts

```diff
@@ -0,0 +1,212 @@
+import {
+  HttpClientTestingModule,
+  HttpTestingController,
+} from '@angular/common/http/testing';
+import { TestBed } from '@angular/core/testing';
+import { OccConfig } from '../../config/occ-config';
+import {
+  CustomerCoupon,
+  CustomerCouponSearchResult,
+  CustomerCouponNotification,
+  CustomerCoupon2Customer,
+} from '../../../model/customer-coupon.model';
+import { OccCustomerCouponAdapter } from './occ-customer-coupon.adapter';
+import { MockOccEndpointsService } from './unit-test.helper';
+import { OccEndpointsService } from '../../services/occ-endpoints.service';
+import { ConverterService } from 'projects/core/src/util/converter.service';
+import { Type } from '@angular/core';
+
+const userId = 'mockUseId';
+
+const couponCode = 'testcouponcode';
+
+const MockOccModuleConfig: OccConfig = {
+  backend: {
+    occ: {
+      baseUrl: '',
+      prefix: '',
+    },
+  },
+};
+
+describe('OccCustomerCouponAdapter', () => {
+  let occCustomerCouponAdapter: OccCustomerCouponAdapter;
+  let httpMock: HttpTestingController;
+  let converter: ConverterService;
```

Comment:
need to add test case `should use convertor`
refer to https://github.com/SAP/cloud-commerce-spartacus-storefront/pull/4137/files#diff-3ce24e75b5602b029869a48d5548952cR253